src/tests/test_integration/test_shopping_cart.py

```
@@ -1,9 +1,16 @@
 # type: ignore
 import pytest
 from datetime import datetime
+from sqlalchemy import create_engine, select
```

Ruff (0.11.9)

`sqlalchemy.create_engine` imported but unused

Remove unused import

(F401)